src/core/components/Paginate/index.js

```diff
@@ -74,9 +75,6 @@ export class PaginateBase extends React.Component {
     if (count === undefined) {
       throw new Error('The count property cannot be undefined');
     }
-    if (currentPage === undefined) {
```

Comment:
Shouldn't the `defaultProps` setting mean this doesn't happen? I forget how `defaultProps` work exactly...

Reply:
Correct, which is why I removed the code. `currentPage` will never be undefined, so this `if` statement is superfluous.

Reply:
I'm wondering what would happen if currentPage was null or 0 due to an error in a parent component. Would be nice to have an error thrown if that will break things.

Reply:
Ah, good point. I'll check that and if it's an issue I'll issue a fix.

Reply:
Guess what? I totally broke pagination with this fix! I'll open a new issue and hopefully it's something I can fix fairly quickly.

Reply:
That's weird, I got pagination working locally when I tried this...